### PR TITLE
Update store setup form styling

### LINKED ISSUE #81: Update store setup step form styling
Form styling needs to be changed to match - p58i-7kf-p2

### Before
screen shot 2018-10-24 at 1 29 48 pm

### After
screen shot 2018-10-24 at 1 30 18 pm

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #81 

This PR adds in the form styling for Step 1 (Store Setup) of the OBW.  p58i-7kf-p2

### Screenshots
screen shot 2018-10-26 at 1 44 10 pm

### Testing
1.  Visit `/wp-admin/admin.php?page=wc-setup&calypsoify=1`
2.  Check that form styling inside white box matches the new ecommerce plan hifi design
3.  Check that "Add address line 2" displays the input box for "Address line 2"

### CODE REVIEW

File: assets/css/calypsoify.css [hunk 1]
```diff
@@ -131,12 +131,134 @@ div.woocommerce-message, .wc-helper .start-container {
...
+}
+.store-address-container .city-and-postcode {
+    margin-bottom: 20px;
+}
+.product-type-container {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+#store_address_2 {
+	display: none;
```
Comment (timmyc) [inline assets/css/calypsoify.css]: Looks like a little bit of indent oddness going on here and down in the next selector too.

File: includes/class-wc-calypso-bridge-admin-setup-wizard.php [hunk 2]
```diff
@@ -134,13 +134,154 @@ public function setup_wizard_content() {
...
+	 * Location, product type, page setup, and tracking opt-in.
+	 */
+	public function wc_setup_store_setup() {
+		$address        = WC()->countries->get_base_address();
+		$address_2      = WC()->countries->get_base_address_2();
+		$city           = WC()->countries->get_base_city();
+		$state          = WC()->countries->get_base_state();
+		$country        = WC()->countries->get_base_country();
+		$postcode       = WC()->countries->get_base_postcode();
+		$currency       = get_option( 'woocommerce_currency', 'GBP' );
```
Comment (timmyc) [inline includes/class-wc-calypso-bridge-admin-setup-wizard.php]: Sound as the pound!

**Comment by joshuatf** [comment]
@timmyc Hmm, the stylesheet it looks like you're missing is actually `/wp-content/plugins/woocommerce/assets/css/wc-setup.css` enqueued by WC.

Which version of WC are you running for this one?  It's possibly related to the weird enqueueing bugs I had before where enqueued scripts/styles were not loading at all.

@justinshreve Can you give this a shot too and let me know if you have any of the same issues?

**Comment by joshuatf** [comment]
@timmyc I just pushed a change that resolves the odd script enqueueing behavior I was having.  Can you pull down the latest and let me know if it's working on your end?

For the console error, this seems to be thrown on every page, so I went ahead and opened an issue in Jetpack - https://github.com/Automattic/jetpack/issues/10474

File: assets/css/calypsoify.css (hunk 1 above)
Comment (joshuatf) [reply assets/css/calypsoify.css]: Thanks!  Didn't show up in my editor at all for some reason.  Reindented and pushed!

**Pull request merged**